static/js/chart/draw.ts

```diff
@@ -427,10 +429,174 @@ function drawLineChart(
   }
 }
 
+
+/**
+ * Draw a group of line charts with different goeId.
+ *
+ * @param id
+ * @param width
+ * @param height
+ * @param dataGroupsDict
+ * @param unit
+ */
+function drawGroupLineChart(
+    id: string,
+    width: number,
+    height: number,
+    dataGroupsDict: {[geoId: string]: DataGroup[];},
+    unit?: string
+) {
+  console.log(dataGroupsDict);
+
+  let dataGroups;
+  for (let geoId in dataGroupsDict) {
+    dataGroups = dataGroupsDict[geoId];
+    break;
+  }
+
+  let minV = 0;
+  let maxV = 0;
+
+  let legendText = dataGroups.map((dataGroup) => dataGroup.label ? dataGroup.label: 'a');
```

Comment:
Please don't carry over the 'a' hack :) In this case, you should always have legend labels to draw.

Reply:
Done